interface/lammps/USER-NEP/pair_NEP.cpp

```diff
@@ -60,6 +60,7 @@ PairNEP::~PairNEP()
   if (allocated) {
     memory->destroy(setflag);
     memory->destroy(cutsq);
+    delete[] type_map;
```

Comment:
There is no `delete[] map`?

Reply:
Pair::map will be deleted by Pair::~Pair()
![image](https://github.com/user-attachments/assets/14ba4aea-e25e-4f78-bf0a-63fe403a32f0)

Reply:
OK